pandas/core/arrays/sparse.py

```diff
@@ -1645,6 +1645,7 @@ def _add_unary_ops(cls):
         cls.__pos__ = cls._create_unary_method(operator.pos)
         cls.__neg__ = cls._create_unary_method(operator.neg)
         cls.__invert__ = cls._create_unary_method(operator.invert)
+        cls.__abs__ = cls._create_unary_method(operator.abs)
```

Comment:
I think SparseArray defines `__abs__`. That can be removed now, right?

Reply:
Correct, removed `__abs__`.